apps/admin-server/src/pages/projects/[project]/widgets/resourceform/[id]/items.tsx

```diff
@@ -1027,6 +1039,152 @@ export default function WidgetResourceFormItems(
+                                                                style={{
+                                                                    padding: "11px",
+                                                                    borderLeft: "4px solid red",
+                                                                    backgroundColor: "#ffdbd7",
+                                                                    borderTopRightRadius: '5px',
+                                                                    borderBottomRightRadius: '5px',
+                                                                    marginTop: '12px',
+                                                                }}
+                                                              >
+                                                                  De geselecteerde vraag heeft nog geen antwoordopties. Voeg deze eerst toe om deze vraag te kunnen tonen op basis van een ander antwoord.
+                                                              </p>
+                                                            ) : (
+                                                              <Select
+                                                                value={field.value}
+                                                                onValueChange={field.onChange}>
+                                                                  <FormControl>
+                                                                      <SelectTrigger>
+                                                                          <SelectValue placeholder="Kies een antwoord" />
+                                                                      </SelectTrigger>
+                                                                  </FormControl>
+                                                                  <SelectContent>
+                                                                      { options.map((o: any) => (
+                                                                        <SelectItem key={o.trigger} value={o.trigger}>{o.titles?.[0]?.key || o.trigger}</SelectItem>
+                                                                      )) }
+                                                                  </SelectContent>
+                                                              </Select>
+                                                            )}
+
+                                                            <FormMessage/>
+                                                        </FormItem>
+                                                      )
+                                                  }}
+                                                />
+                                              )}
+                                          </>
+                                        )}
```

Comment:
This entire block is repeated multiple times, can this be refactored so we don't have to maintain it in multiple locations?